psp-bugos/psp-bugos.csproj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
```

Comment:
When it comes to properties,`<WarningsAsErrors>nullable</WarningsAsErrors>` could be added.

Reply:
added `<WarningsAsErrors>nullable</WarningsAsErrors>` line